src/graphql/typeDefs/dataTypes/milestoneInput.graphql

```diff
@@ -1,11 +1,17 @@
 input MilestoneCreateInput {
+  "Title of milestone"
   title: String!
-  description: String!
-  dueDate: Date!
+  "Description of this milestone"
+  description: String
+  "The date associated with the milestone"
+  dueDate: Date
```

Comment:
Shouldn't this be required?

Reply:
Updated in commit cf4bfb72c7969d271fc7d2983b427d7a9a2de08a